platform/src/main/resources/anchor-docker-compose-config.yaml

```diff
@@ -284,10 +284,9 @@ data-spring-jdbc-sqlite:
   spring.datasource.username: admin
   spring.datasource.password: admin
   spring.mvc.converters.preferred-json-mapper: gson
-  spring.liquibase.enabled: false    # certain features that liquibase uses (ex: addForeignKeyConstraint) are not supported by sqlite
+  spring.flyway.enabled: false    # certain features that liquibase uses (ex: addForeignKeyConstraint) are not supported by sqlite
```

Comment:
Do we no longer support liquid base?

Reply:
yeah, liquibase was removed a couple months ago